### PR TITLE
Update manifest

### PR DESCRIPTION
**Pull request by jiqiang90**

### CODE REVIEW

**Review by jamesbayly — COMMENTED** [review]
Few things
- Where is our documentation for the new chainTypes file
- Is it `chaintypes` or `chainTypes` in the manifest?
- Where does a user find the genesis hash of a network?

**Comment by jiqiang90** [comment]
> Few things
> 
> * Where is our documentation for the new chainTypes file
> * Is it `chaintypes` or `chainTypes` in the manifest?
> * Where does a user find the genesis hash of a network?

- chainType is a set include types , typesAlias, typesBundle, typesChain, and typesSpec. It is same structure as how we defined in manifest v0.0.1, but now it is been moved into a single file. 
- Sure it is the `chaintypes` https://github.com/subquery/subql-starter/blob/b788d5416f446f9d5335e122a4a0a10cbc331658/project.yaml#L15.
- If you run `subql migrate` or `subql init` with latest dev cli, you can see it will automatically fetch the genesis hash from the endpoint, and it is placed in the new manifest for user. Also, we are expecting user able to manually find genesis hash (block 0) from the explorer (polkadotjs or subscan)

**Review by jiqiang90 — COMMENTED** [review]
https://github.com/subquery/subql/pull/594/commits/146b363610e7de34b89c0c2d0177fcd35163b293
Is there a particular reason for changing the title?
`Data sources` means which we will include all custom data sources here, and Moonbeam is just one of them.
Otherwise, on the top level of our side menu will have a list of partners we supported.

File: docs/create/manifest.md [hunk 1]
```diff
@@ -2,24 +2,62 @@
...
+name: example-project # Provide the project name
+version: 1.0.0  # Project version
+description: '' # Description of your project
+repository: 'https://github.com/subquery/subql-starter' # Git repository address of your project
+
+schema:
+file: ./schema.graphql # The location of your GraphQL schema file
+
+network:
+genesisHash: '0x91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3' # Genesis hash of the network
```
Comment (ianhe8x) [inline docs/create/manifest.md:20]: indent issue here

Comment (jiqiang90) [reply docs/create/manifest.md]: fixed

**Review by jamesbayly — APPROVED** [review]

**Review by ianhe8x — APPROVED** [review]

**Pull request merged**